lib/wallet/walletdb.js

```diff
@@ -196,6 +197,12 @@ class WalletDB extends EventEmitter {
       this.state.height,
       this.state.startHeight);
 
+    const clientInfo = await this.client.getInfo();
```

Comment:
This isn't used for anything in walletdb at this point, that looks to be a part of https://github.com/bcoin-org/bcoin/pull/751 rather.